### PR TITLE
Setup homepage file, style and test

### PR DESCRIPTION
**Pull request by jihaneH**
Branch created to provide feedback on a past project

### CODE REVIEW

**Review by jihaneH — COMMENTED** [review]
🛑 Make linter please 🛑

**Review by jihaneH — COMMENTED** [review]
Please fix the linters
🛑 🛑 🛑 🛑 🛑 🛑 🛑 🛑 
Capture d’écran 2021-11-29 à 14 01 33

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (jihaneH) [inline .github/workflows/linters.yml:48]: Just a comment for test

File: .hintrc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
```
Comment (jihaneH) [inline .hintrc:16]: Just a test

**Review by jihaneH — COMMENTED** [review]
Good job ! 👍 👋

File: .github/workflows/linters.yml (hunk 1 above)
Comment (jihaneH) [reply .github/workflows/linters.yml:48]: ok

File: .hintrc (hunk 2 above)
Comment (jihaneH) [reply .hintrc:16]: no problem

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @jihaneH, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] To keep your repo professional, your Pull Request should always have a small description of the changes made in that branch, for instance, you could say that this branch was created to provide feedback on a past project made by you.
- [ ] According to the requirements, this is how you should communicate in a code review:

image
- [ ] Keep in mind that your comments should actually have an improvement suggestion, shouldn't be only a test. According to the requirements, you should make at least 2 comments in your review.


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.projects-grid img {
+  width: 300px;
+  margin: 10px;
+}
```
Comment (jihaneH) [inline css/style.css:95]: Please compare your design and the template design and try to have marge and width to match the template

File: index.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+          <a href="#projects">Work</a>
+        </li>
+       <li>
+          <a href="#contact">Contact</a>
+        </li>
+      </ul>
+      </nav-bar>
+    </nav>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Hey We are Mani & Jihane</h1>
```
Comment (jihaneH) [inline index.html:30]: Please compare your design and the template design and try to have text align to match the template

Your work :
Capture d’écran 2021-11-29 à 17 37 45

The template :
Capture d’écran 2021-11-29 à 17 39 08

**Review by jihaneH — COMMENTED** [review]
Good job! 👍 👏 👏
There is some issue doesn't been addressed yet.

**Review by andersonlebon — APPROVED** [review]
Hi @jihaneH 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by jihaneH** [comment]
icegif-1433

**Pull request merged**